bin_tests/src/bin/crashtracker_bin_test.rs

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present Datadog, Inc.
+
+#[cfg(not(unix))]
+fn main() {}
+
+#[cfg(unix)]
+fn main() -> anyhow::Result<()> {
+    unix::main()
+}
+
+#[cfg(unix)]
+mod unix {
+    use std::env;
+
+    use datadog_crashtracker::{
+        self as crashtracker, CrashtrackerConfiguration, CrashtrackerMetadata,
+    };
+    use datadog_profiling::exporter::Tag;
+
+    #[inline(never)]
+    unsafe fn deref_ptr(p: *mut u8) {
+        *std::hint::black_box(p) = std::hint::black_box(1);
+    }
+
+    pub fn main() -> anyhow::Result<()> {
+        let mut args = env::args().skip(1);
+        let output_filename = args
+            .next()
+            .ok_or(anyhow::anyhow!("Unexpected number of arguments"))?;
+        let receiver_binary = args
+            .next()
+            .ok_or(anyhow::anyhow!("Unexpected number of arguments"))?;
+        crashtracker::init(
+            CrashtrackerConfiguration {
+                create_alt_stack: true,
+                endpoint: Some(datadog_profiling::exporter::Endpoint {
+                    url: ddcommon::parse_uri(&format!("file://{}", output_filename))?,
+                    api_key: None,
+                }),
+                path_to_receiver_binary: receiver_binary,
+                resolve_frames: crashtracker::CrashtrackerResolveFrames::ExperimentalInProcess,
```

Comment:
This may crash.  Might be better to use `Never` for now

Reply:
It in fact, does not pass the tests on the CI with `ExperimentalInProcess`, crashes the process in debug mode and produces an invalid json when receiving the crash report in release mode.